includes/payment-methods/class-wc-stripe-upe-payment-method-link.php

```diff
@@ -33,8 +33,9 @@ public function __construct() {
 	 * @return bool
 	 */
 	public static function is_link_enabled() {
+
 		// Assume Link is disabled if UPE is disabled.
-		if ( ! WC_Stripe_Feature_Flags::is_upe_checkout_enabled() ) {
+		if ( ! self::is_available() || ! WC_Stripe_Feature_Flags::is_upe_checkout_enabled() ) {
```

Comment:
Similarly here, if we prevent Link from being available from `get_upe_available_payment_methods`, I don't think it will ever be enabled, so we can remove the extra check and rely on the next line to check if it's enabled.

Reply:
changed in [0ea817f](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2452/commits/0ea817f020274f678061c27bbd199a874a2578e8) . thx